configure

```diff
@@ -370,7 +370,7 @@ if test $enable_prof -eq 1
 then
     echo -n "checking whether to enable profiling: "
     case "${overs}" in
-	4.08.[1-9]*|4.09.*|4.[1-9]*)
+	4.08.[1-9]*|4.09.*|4.[1-9]*|5*)
```

Comment:
One last thing: could you make it `[5-9]*|[1-9][0-9]*)` instead (for future-proofing — even if it's likely to be obsolete when we reach such versions of OCaml :wink: )?

Reply:
Sure 😀